apus_token/main.lua

```diff
@@ -22,14 +22,18 @@ Token = require('token')
 Allocator = require('allocator')
 Distributor = require('distributor')
 
-AO_MINT_PROCESS = "LPK-D_3gZkXtia6ywwU1wRwgFOZ-eLFRMP9pfAFRfuw"
-APUS_STATS_PROCESS = "zmr4sqL_fQjjvHoUJDkT8eqCiLFEM3RV5M96Wd59ffU"
+require('config')
 
 -- Function to verify if a message is a mint report from AO Mint Process
 local function isMintReportFromAOMint(msg)
   return msg.Action == "Report.Mint" and msg.From == AO_MINT_PROCESS
 end
 
+-- Function to verify if a message can be processed with the target handler
+local function isMintBackupAuthorized(msg)
+  return msg.Action == "Mint.Backup" and (msg.From == ao.env.Process.Owner or msg.From == ao.id)
```

Comment:
ao.Owner

Reply:
> ao.Owner

I have tried but no response in aos console.